packages/storybook/stories/password-creation-input.mdx

```diff
@@ -20,7 +20,7 @@ The password creation input is a pattern that allows the users to create a passw
 - Use to display all the conditions necessary to create or edit a password.
 
 ### When not to use
-Do not use for anything else than a password creation or edition.
+- Do not use for anything else than a password creation or edition.
```

Comment:
Voir commentaire dans la section "_When to use_" à propos de "_edit_" vs "_change_".